packages/web-components/fast-components/src/flipper/flipper.styles.ts

```diff
@@ -27,7 +27,7 @@ export const FlipperStyles = css`
         align-items: center;
         margin: 0;
         position: relative;
-        fill: ${accentForegroundCutRestBehavior.var};
+        fill: currentColor;
```

Comment:
```suggestion
        fill: currentcolor;
```

Reply:
+1 to all of these, but it does look like the browser normalizes here, which is likely why it's also correct upon testing for @brookdozer :).

Reply:
Updated to all to currentcolor.